wcfsetup/install/files/lib/system/cronjob/RefreshSearchRobotsCronjob.class.php

```diff
@@ -15,15 +16,23 @@
  * @package	WoltLabSuite\Core\System\Cronjob
  */
 class RefreshSearchRobotsCronjob extends AbstractCronjob {
+	/**
+	 * URL to the spider list.
+	 */
+	const SPIDER_LIST_URL = 'http://assets.woltlab.com/spiderlist/typhoon/list.xml';
```

Comment:
Consider adding an access modifier here. I suggest `private`:
```suggestion
	private const SPIDER_LIST_URL = 'http://assets.woltlab.com/spiderlist/typhoon/list.xml';
```

Reply:
What is the advantage of this, except that it can no longer be read from the outside? This is not a critical value.

Reply:
> except that it can no longer be read from the outside?

Exactly that. Anything that is `public` or `protected` is part of the public API. For better flexibility we should strive to not expose stuff that does not need to be exposed.